src/pages/home/report/ReportActionItemMessageEdit.js

```diff
@@ -93,17 +94,28 @@ function ReportActionItemMessageEdit(props) {
     const {translate} = useLocalize();
     const {isKeyboardShown} = useKeyboardState();
     const {isSmallScreenWidth} = useWindowDimensions();
+    const isMobileSafari = Browser.isMobileSafari();
 
-    const [draft, setDraft] = useState(() => {
+    const getInitialDraft = () => {
         if (props.draftMessage === props.action.message[0].html) {
             // We only convert the report action message to markdown if the draft message is unchanged.
             const parser = new ExpensiMark();
             return parser.htmlToMarkdown(props.draftMessage).trim();
         }
         // We need to decode saved draft message because it's escaped before saving.
         return Str.htmlDecode(props.draftMessage);
-    });
-    const [selection, setSelection] = useState({start: 0, end: 0});
+    };
+
+    const getInitialSelection = () => {
+        const length = getInitialDraft().length;
```

Comment:
Can we move this calculation after the `isMobileSafari` if condition? We don't need to calculate it for safari, so let's calculate it only for other platforms

Reply:
Moved after the isMobileSafari check.